src/main/java/org/isf/utils/pagination/PageInfo.java

```diff
@@ -21,11 +21,14 @@
  */
 package org.isf.utils.pagination;
 
+import org.springframework.data.domain.Page;
+
 public class PageInfo {
 	int size;
 	int page;
 	int nbOfElements;
-	long totalCount;
+	long totalNbOfElements;
+	int totalPages;
```

Comment:
is there any reason why `totalNbOfElements` is `long` instead of `int` like the others?
what is the difference with `nbOfElements`?

Reply:
No totalNbOfElements can be an int. difference is nbOfElements is the nomber of elements of the page and totalNbOfElements  is le numbers of all the elements that we have in database which match the param of the filter.

Reply:
so why `totalNbOfElements` is long?

Reply:
Because it is the type of data return by the method getTotalElements of the Page object.